### ISSUE TITLE
Resolve meta accessibility violations

### ISSUE THREAD

**Issue by lawhorkl** [uclibs/scholar_uc#1588]
### Descriptive summary

Our recent accessibility scan of Scholar 3 reveals 105 total accessibility issues, 12 of these are issues with the meta tags in Scholar not disabling text scaling and zooming.

We need to resolve them before we push to prod.

To new developers: Do not be intimidated by the list length, it is most likely the case that one or two changes to the head partials will resolve these.

A good tool while working on this is https://achecker.ca/checker/index.php to allow you to make sure your changes solve the issue.

The following is a list of all meta tag accessibility issues.

* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044371
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044394
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044401
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044409
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044417
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044587
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044763
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044954
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045089
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045260
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045411
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045561
